### ISSUE TITLE
[Accessibility] Dialog has empty value for "aria-describedby" when no subtext is provided.

### ISSUE THREAD

**Issue by shadyysf** [OfficeDev/office-ui-fabric-react#1088]
### Bug Report

- __Package version(s)__: 1.7.2
- __Browser and OS versions__: All

#### Describe the issue:

Dialog container div has no valid value for "aria-describedby" if no subtext property is provided.

#### Actual behavior:
 Container Div dialog will have no value for "aria-describedby" if no subtext property is provided. This violates the aria restriction that aria-{something} attributes must have valid values. https://dequeuniversity.com/rules/axe/2.1/aria-valid-attr-value

`<div role="dialog" aria-labelledby="Dialog122-title" aria-describedby=""></div>`

Looking at the code for the fabric  [Dialog](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx)

` <Popup
            role={ isBlocking ? 'alertdialog' : 'dialog' }
            ariaLabelledBy={ title ? id + '-title' : '' }
            ariaDescribedBy={ subText ? id + '-subText' : '' }
            onDismiss={ onDismiss }
          >`
#### Expected behavior:

If no subtext property is defined, we should not have an "aria-describedby" property at all and not put one with an empty value.

**Comment by gitjain** [comment]
Updating the status as solution has been merged.
